build/gulpfile.vscode.linux.js

```diff
@@ -15,12 +15,35 @@ const util = require('./lib/util');
 const packageJson = require('../package.json');
 const product = require('../product.json');
 
+const linuxPackageRevision = Math.floor(new Date().getTime() / 1000);
+
+const flatpakManifest = {
+	appId: 'com.microsoft.CodeOSS',
+	sdk: 'org.freedesktop.Sdk',
+	runtime: 'org.freedesktop.Sdk',
+	runtimeVersion: '1.4',
```

Comment:
This is the version of the runtime bundle that the application needs. It is used both for running (`org.freedesktop.Platform`) and building (`org.freedesktop.Sdk`). The current version is `1.4`. Setting this ensures that the application will be always ran using an userland which conforms exactly to the ABI exposed by the 1.4 version of the runtime.